### PR TITLE
[docs] add certgen install/uninstall in quickstart, other minor updates

### PR DESCRIPTION
**Pull request by chauhanshubham**

### CODE REVIEW

File: docs/user/QUICKSTART.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Install the Gateway API CRDs:
 kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/v0.2.0-rc2/gatewayapi-crds.yaml
 ```
 
+Run the certgen Job required for Envoy Gateway:
```
Comment (arkodg) [inline docs/user/QUICKSTART.md:15]: certgen should get installed as part of `install.yaml`, is that not happening ?

Comment (chauhanshubham) [reply docs/user/QUICKSTART.md:15]: right, I think this happened for me because I was running the eg binary locally.

Comment (danehans) [reply docs/user/QUICKSTART.md]: @chauhanshubham you are correct, the issue is related to running EG locally. I would like us to develop a solution to properly run EG locally after the v0.2.0. I think it's a good idea to comment out the run-kube-local target until EG is fully functional when running locally.

File: docs/design/gatewayapi-translator.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 ## Assumptions
-- initially target core conformance features only, to be followed by extended conformance features
+- Initially target core conformance features only, to be followed by extended conformance features
 
 ## Inputs and Outputs
 
 The main inputs to the Gateway API translator are:
 - the GatewayClass to process
-- Gateways, HTTPRoutes, Services, Secrets
+- Gateways, HTTPRoutes, Services, Secrets, ReferenceGrants
```
Comment (danehans) [inline docs/design/gatewayapi-translator.md:10]: @chauhanshubham TLSRoutes should be added as part of your TLS pass through work.

**Pull request closed without merging**